src/github.com/getlantern/flashlight/client/handler.go

```diff
@@ -32,39 +34,59 @@ func (client *Client) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 // connetion and starts piping the data over a new net.Conn obtained from the
 // given dial function.
 func (client *Client) intercept(resp http.ResponseWriter, req *http.Request) {
-	if req.Method != "CONNECT" {
+	var err error
+
+	// intercept can only by used for CONNECT requests.
+	if req.Method != httpConnectMethod {
 		panic("Intercept used for non-CONNECT request!")
 	}
 
-	// Hijack underlying connection
-	clientConn, _, err := resp.(http.Hijacker).Hijack()
-	if err != nil {
+	// Hijacking underlying connection.
```

Comment:
Nit: I think the imperative "hijack" is more idiomatic for comments.  I wouldn't have bothered to make this remark on a new comment, but I think changing things should only be done if it's clearly for the better.